main.lua

```diff
@@ -346,11 +346,11 @@ local function GridLayout(unsortedFrames)
                 local s1 = v.state
                 for i,e in ipairs(frames) do
                     local s2 = e.state
-                    if (s2.isNumber and s1.isNumber) and (s2.value < s1.value) then
+                    if (not s2.miss and not s1.miss) and (s2.amount < s1.amount) then
                         tinsert(frames, i, v)
                         count = count + 1
                         break
-                    elseif (s1.isString ~= s2.isString) and (missPrio or s1.isNumber) then
+                    elseif (s1.miss ~= s2.miss) and (missPrio or not s1.miss) then
```

Comment:
this never executes unless miss info is provided
also looking at this led me to finding a bug I introduced here
to fix it I believe it has to go like this
```suggestion
                    elseif (s1.miss ~= s2.miss) and ((s1.miss and missPrio) or not s1.miss) then
```